scripts/_1_1_3_4d_probe_responses.py

```diff
@@ -0,0 +1,264 @@
+            "psTarget",
+            "PS",
+            "ps_kdma_aligned_set1",
+            "ps_kdma_baseline_set1",
+            "set1_aligned_probe_responses",
+            "set1_baseline_probe_responses",
+        ),
+        (
+            "July2025-PS2-eval",
+            "psTarget",
+            "PS",
+            "ps_kdma_aligned_set2",
+            "ps_kdma_baseline_set2",
+            "set2_aligned_probe_responses",
+            "set2_baseline_probe_responses",
+        ),
+        (
+            "July2025-PS3-eval",
+            "psTarget",
+            "PS",
+            "ps_kdma_aligned_set3",
+            "ps_kdma_baseline_set3",
+            "set3_aligned_probe_responses",
+            "set3_baseline_probe_responses",
+        ),
+        (
+            "July2025-SS1-eval",
+            "ssTarget",
+            "SS",
+            "ss_kdma_aligned_set1",
+            "ss_kdma_baseline_set1",
+            "set1_aligned_probe_responses",
+            "set1_baseline_probe_responses",
+        ),
+        (
+            "July2025-SS2-eval",
+            "ssTarget",
+            "SS",
+            "ss_kdma_aligned_set2",
+            "ss_kdma_baseline_set2",
+            "set2_aligned_probe_responses",
+            "set2_baseline_probe_responses",
+        ),
+        (
+            "July2025-SS3-eval",
+            "ssTarget",
+            "SS",
+            "ss_kdma_aligned_set3",
+            "ss_kdma_baseline_set3",
+            "set3_aligned_probe_responses",
+            "set3_baseline_probe_responses",
+        ),
+    ]
+
+    processed = 0
+    total_docs_created = 0
+
+    for doc in documents:
+        processed += 1
+        pid = doc.get("pid", "unknown")
+        print(f"Processing document {processed} of {doc_count} (PID: {pid})")
+
+        for (
+            scenario,
+            target_field,
+            kdma_type,
+            aligned_kdma_field,
+            baseline_kdma_field,
+            aligned_probes_field,
+            baseline_probes_field,
+        ) in scenarios:
+            aligned_probes_raw = doc.get(aligned_probes_field, [])
+            aligned_probes_parsed = parse_probe_responses(aligned_probes_raw, kdma_type)
+
+            baseline_probes_raw = doc.get(baseline_probes_field, [])
+            baseline_probes_parsed = parse_probe_responses(
+                baseline_probes_raw, kdma_type
+            )
+
+            aligned_doc = {
+                "evalNumber": 11,
+                "evalName": "Phase 2 4D Experiment",
+                "pid": pid,
+                "scenario": scenario,
+                "alignment_target": str(doc.get(target_field)), 
+                "adm_name": doc.get("admName"),  
+                "synthetic": True, 
+                "probe_ids": [
+                    list(pr.keys())[0] for pr in aligned_probes_parsed
+                ],  
+                "probe_responses": aligned_probes_parsed,
+                "results": {
+                    "ta1_session_id": pid,
```

Comment:
This seems wrong.  Do we have the TA1 session ID?  Or is this some surrogate if we don't have it?  Maybe it's just worth omitting if we don't have it.